Mjml.Net/Components/Body/MsoButtonComponent.cs

```diff
@@ -0,0 +1,153 @@
+﻿using Mjml.Net.Extensions;
+using Mjml.Net.Types;
+
+namespace Mjml.Net.Components.Body
+{
+    public partial class MsoButtonComponent : ButtonComponent
+    {
+        private Dictionary<string, string> BorderstyleAdapter = new()
```

Comment:
This should be either static or just use a switch statement, it is probably less code.

Reply:
Refactored into method with switch statement